doc/source/ray-air/tuner.rst

```diff
@@ -0,0 +1,216 @@
+.. _air-tuner:
+
+Configuring Hyperparameter Tuning
+=================================
+
+The Ray AIR :class:`Tuner <ray.tune.Tuner>` is the recommended way to tune hyperparameters in Ray AIR.
+
+.. image:: images/tuner.svg
+    :align: center
+
+`Tuner` provides an interface that works with AIR Trainers to perform distributed
+hyperparameter tuning. `Tuner` is part of Ray Tune and provides
+a variety of state-of-the-art hyperparameter tuning algorithms for optimizing model
+performance.
+
+This page will cover basic usage examples of a Tuner. If you are interested in
+reading more, please take a look at the :ref:`Ray Tune documentation <tune-main>`.
+
+Key Concepts
+------------
+
+There are a couple key concepts for using a Tuner:
+
+* First, you need to specify the hyperparameters you want to tune in a `search space`.
```

Comment:
Remove first. Not needed.

 * A specific set of hyperparameters in a `search space.`
 * A `search algorithm` to ...
 * A set of arguments (`search space`, `search algorithm`, `scheduler`, and `Trainer`) passed to the `Tuner`
 *  Each individual hyperparater evalution run is called a `trial`
 * The `Tuner` returns its results in a `ResultGrid` object.